src/FlywheelCore.sol

```diff
@@ -134,7 +136,7 @@ contract FlywheelCore is Auth {
         require(strategyState[strategy].index == 0, "strategy");
         strategyState[strategy] = RewardsState({
             index: ONE,
-            lastUpdatedTimestamp: uint32(block.timestamp)
+            lastUpdatedTimestamp: block.timestamp.safeCastTo32()
```

Comment:
i still think using uint32 is a bad idea for timestamps, the system will be bricked in ~83 years

Reply:
I don't think we need to plan that far ahead tbh, 83 years ago neither computers nor the internet existed

Reply:
bruh ok lol

Reply:
also mainframe computers were around since the 30s